XRTK.SDK/Packages/com.xrtk.sdk/Features/Input/Handlers/ManipulationHandler.cs

```diff
@@ -284,9 +288,20 @@ public Vector2 NudgeConstraints
         /// </summary>
         private Vector2 lastPositionReading = Vector2.zero;
 
-        private bool isPressed = false;
+        protected bool isPressed = false;
```

Comment:
Let's make these protected fields [properties instead](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/field).

```
X DO NOT provide instance fields that are public or protected.

You should provide properties for accessing fields instead of making them public or protected.
```

Reply:
After reviewing our internal changes I noticed we were not using these changes at all.

Please revert these field changes.